open-spatial-lab/dextra
# Dot map

## Overview

This PR creates a dot map that uses the geo mark. 


### Demo

<img width="1171" alt="image" src="https://github.com/open-spatial-lab/dextra/assets/102711675/2bafa542-6716-45ff-91d5-ccd2bf68016b">


### Notes
Changed mark name from "choropleth" to "geo"

## Testing Instructions

- Run "pnpm dev: components" in terminal
- Go to "http://localhost:6006/"
- Find and Click "Dot Map" on the left hand side menu. 
- Expected outcome: grey land map and red bubbles showing the earthquake intensity. 
- Click any of the red bubble in the plot, the link should go to USGS’s live API.
